### DISCUSSION TITLE
Bring back smaller streaks widgets

### DISCUSSION THREAD

**Discussion by sooshii** [iSoron/uhabits#1625] [Feature Requests]
I'm quite upset that smaller widgets are no longer available and was wondering if it was possible to bring them back. As you can see in my screenshot the widget at the top is smaller and more preferable (at least for me), but new widgets do not size down like the old ones.
They can only go as small as 2 vertically ![screenshot](https://user-images.githubusercontent.com/73917729/212778655-13ad1097-b40f-47d6-833e-10db2ffefebc.jpg)

**Comment by ezio-melotti** [comment]
I ran into the same problem with the Target widget, and reported it here:
* https://github.com/iSoron/uhabits/issues/2118

Your screenshot confirms that the Streaks widget also works as 4x1.

**Reply by ZornHadNoChoice** [reply]
The OP's problem is actually solved. Here is a screenshot from the latest version:
![Screenshot_2025-03-15-07-42-13-986_com mi android globallauncher-edit](https://github.com/user-attachments/assets/65f840e0-1f91-480e-824c-f370494b02db)
